.github/workflows/prerelease.yml

```diff
@@ -0,0 +1,17 @@
+# This needs to be added in the master branch to show the workflow fir all branches on GitHub
```

Comment:
typo: ...show the workflow **for** all...

Reply:
Fixed!